README.md

```diff
@@ -4,11 +4,13 @@
 > This is a Book Store web app where we can choose a book, a category and see the writer and our reading progress.
 
 # GIF/Screenshot
-![](gif-2.gif)
+![](gif-3.gif)
```

Comment:
[OPTIONAL] The screenshot is not displayed on the Readme file. I recommend removing `-` from the name of the screenshot.

Reply:
Thank you @medaminedev66 . I have updated it.